cfn-resources/private-endpoint-aws/mongodb-atlas-privateendpointaws.json

```diff
@@ -1,74 +1,74 @@
 {
-    "typeName": "MongoDB::Atlas::PrivateEndpointAWS",
-    "description": "Creates one private endpoint for the specified cloud service provider. At this current version only AWS is supported",
-    "sourceUrl": "https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/cfn-resources/private-endpoint-aws",
-    "documentationUrl": "https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/cfn-resources/private-endpoint-aws/README.md",
-    "tagging": {
-        "taggable": false
+  "typeName": "MongoDB::Atlas::PrivateEndpointAWS",
+  "description": "Creates one private endpoint for the specified cloud service provider. At this current version only AWS is supported",
+  "sourceUrl": "https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/cfn-resources/private-endpoint-aws",
+  "documentationUrl": "https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/cfn-resources/private-endpoint-aws/README.md",
+  "tagging": {
+    "taggable": false
+  },
+  "required": [
+    "ProjectId",
+    "EndpointServiceId"
+  ],
+  "properties": {
+    "Profile": {
+      "type": "string",
+      "description": "The profile is defined in AWS Secret manager. See [Secret Manager Profile setup](../../../examples/profile-secret.yaml)",
+      "default": "default"
     },
-    "required": [
-        "ProjectId",
-        "EndpointServiceId"
-    ],
-    "properties": {
-        "Profile": {
-            "type": "string",
-            "description": "The profile is defined in AWS Secret manager. See [Secret Manager Profile setup (../../../examples/profile-secret.yaml)",
```

Comment:
All indentation is due to pre-commit hook. Only thing that is really changed is this line